### ISSUE TITLE
Update Performance documentation

### ISSUE THREAD

**Issue by harpsicord86** [ClosedXML/ClosedXML#1309]
Hi folks,

I would seriously change your documentation on performance reading from this:
https://github.com/ClosedXML/ClosedXML/wiki/Where-to-use-the-using-keyword

Because you've removed the IDisposable interface, it means that your documented practice on speeding up performance is no longer valid.

Thanks

_Originally posted by @harpsicord86 in https://github.com/ClosedXML/ClosedXML/issues/1045#issuecomment-554328702_

**Comment by igitur** [comment]
True. You can help out by submitting a PR for the documentation update at https://github.com/ClosedXML/ClosedXML-wiki .
